superset/db_engine_specs/presto.py

```diff
@@ -949,11 +949,7 @@ def get_create_view(
             sql = f"SHOW CREATE VIEW {schema}.{table}"
             try:
                 cls.execute(cursor, sql)
-                polled = cursor.poll()
```

Comment:
@Thelin90 let me test this using Presto to ensure this works.

Reply:
I confirmed, via the [/extra_table_metadata/\<int:database_id>/\<table_name>/\<schema>/](https://github.com/apache/superset/blob/9432c62ec168f449e4ba71112db3c6fc14365710/superset/views/core.py#L2100) route—which is invokes said method for views, that the method works as expected without the polling.